Drivers/Src/MotorController.c

```diff
@@ -11,13 +12,71 @@
 #define MASK_CC_ERR 1<<2 //checks velocity on 2 bit
 #define MASK_OVER_SPEED_ERR 1<<24 //check if motor overshot max RPM on 24 bit
 
+static OS_SEM	MotorController_MailSem4;
+static OS_SEM	MotorController_ReceiveSem4;
+static float CurrentVelocity = 0;
+
+uint16_t Motor_FaultBitmap = 0;
```

Comment:
Would be more readable if this was initialized to `T_NONE` instead of `0` to be consistent with `assertTritiumError()`